src/commands/supplement/tags/write.ts

```diff
@@ -0,0 +1,92 @@
+import {Command, Flags} from '@oclif/core'
+import {ExecJSON, ProfileJSON} from 'inspecjs'
+import fs from 'fs'
+
+export default class WriteTags extends Command {
+    static usage = 'supplement tags write -i <input-hdf-or-profile-json> (-f <input-tags-json> | -d <tags-json>) [-o <output-hdf-json>]'
+
+    static description = 'Overwrite the `tags` attribute in a given Heimdall Data Format or InSpec Profile JSON file and overwrite original file or optionally write it to a new file'
+
+    static summary = 'Tags data can be either a Heimdall Data Format or InSpec Profile JSON file. See sample ideas at https://github.com/mitre/saf/wiki/Supplement-HDF-files-with-additional-information-(ex.-%60tags%60,-%60target%60)'
+
+    static examples = [
+      'saf supplement tags write -i hdf.json -d \'[[{"a": 5}]]\'',
+      'saf supplement tags write -i hdf.json -f tags.json -o new-hdf.json',
+      'saf supplement tags write -i hdf.json -f tags.json -o new-hdf.json -c "V-000001',
+    ]
+
+    static flags = {
+      help: Flags.help({char: 'h'}),
+      input: Flags.string({char: 'i', required: true, description: 'An input HDF or profile file'}),
+      tagsFile: Flags.string({char: 'f', exclusive: ['tagsData'], description: 'An input tags-data file (can contain JSON that matches structure of tags in input file(HDF or profile)); this flag or `tagsData` must be provided'}),
+      tagsData: Flags.string({char: 'd', exclusive: ['tagsFile'], description: 'Input tags-data (can contain JSON that matches structure of tags in input file(HDF or profile)); this flag or `tagsFile` must be provided'}),
+      output: Flags.string({char: 'o', description: 'An output file that matches structure of input file (otherwise the input file is overwritten)'}),
+      controls: Flags.string({char: 'c', description: 'The id of the control whose tags will be extracted', multiple: true}),
+    }
+
+    async run() {
+      const {flags} = await this.parse(WriteTags)
+
+      const input: ExecJSON.Execution | ProfileJSON.Profile = JSON.parse(fs.readFileSync(flags.input, 'utf8'))
+
+      const output: string = flags.output || flags.input
+
+      let tags: any
```

Comment:
no 'any' if at all possible.  if you need to have something there make it 'unknown'.  any more or less disables the type system which is why we try to avoid using it.  however, i'm fairly sure we know the types here.  if it's an execjson, then it's be ExecJson.whatever_tag_was_called[][], and for profile jsons, then it'd be ProfileJson.whatever_tag_was_called[].